packages/react/src/components/DotcomShell/README.md

```diff
@@ -0,0 +1,83 @@
+# DotcomShell
+
+> The DotcomShell component includes the `Masthead`, `Footer`, and `Content`
+> components, all wrapped in a UI shell using Carbon's grid.
+
+## Getting started
+
+The `DotcomShell` uses Carbon's `Content` component. All user-provided content
+should be placed here. This can be anything from HTML to a React component.
+Here's a quick example to get you started.
+
+### Content
+
+```javascript
+import React from 'react';
+import { Button, StructuredListBody } from 'carbon-components-react';
+
+const content = (
+  <>
+    <p>Your content here (probably include something nicer than this! 😄)</p>
+    <Button />
+    <StructuredListBody />
+  </>
+);
+
+export default content;
+```
+
+### DotcomShell
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { DotcomShell } from '@carbon/ibmdotcom-react';
+import '@carbon/ibmdotcom-styles/scss/components/masthead/masthead.scss';
+import '@carbon/ibmdotcom-styles/scss/components/footer/footer.scss';
+import content from 'content';
+
+function App() {
+  return (
+    <DotcomShell navigation={navigation} footerType="short">
+      {content}
+    </DotcomShell>
+  );
+}
+```
+
+> 💡 > And don't forget to import the DotcomShell styles from
+> [@carbon/ibmdotcom-styles](/packages/styles).
+
+## Data and content
+
+| Name          | Description     |
+| ------------- | --------------- |
+| `children`    | User content    |
+| `footer`      | Footer type     |
```

Comment:
@kennylam for footer and navigation can you add links to the corresponding READMEs?
cc: @photodow @annawen1

Reply:
it's already in there
![image](https://user-images.githubusercontent.com/909118/65338291-da44d700-db97-11e9-804b-ed4939d6adc4.png)